kube_controller_manager/CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## Unreleased
 
+***Changed***:
```

Comment:
nit: I think the `Added` changelog would be better than `Changed`, since `Changed` is normally saved for incompatible/breaking changes.

Reply:
thanks for the insight, updated!